src/BinaryChart.js

```diff
@@ -55,7 +57,8 @@ export default class BinaryChart extends Component {
     shouldComponentUpdate(nextProps) {
         if (
             this.props.symbol !== nextProps.symbol ||
-                this.props.type !== nextProps.type                  // this might break when we have more chart type
+                this.props.type !== nextProps.type ||                 // this might break when we have more chart type
+                this.props.pipSize !== nextProps.pipSize
```

Comment:
You're, I must be drunk by then, there's a bug when you switch symbol in nextgen the chart pipsize is gone, must some other reason, I'll try fix it